Annotato/Annotato/Persistence/DocumentSharesPersistenceManager.swift

```diff
@@ -20,6 +20,14 @@ class DocumentSharesPersistenceManager {
         return localSharedDocument
     }
 
+    func deleteDocumentShare(document: Document, recipientId: String) async -> Document? {
```

Comment:
Am I right to say that this does not actually delete the document PDF from local storage? Should we handle that?